Docs for using Steam SDK

arthur740212 opened the pull request.

arthur740212 committed:
doc(steam) : readme_steam.md

Jason-J-PEW reviewed (approved):
The EOS Dev portal changes frequently enough that we may not want to give instructions on how to set up the Identity Provider. And instead only link to their documentation on how to do it. Other than that, it looks good.

Note: I only read through the content, I don't know about the formatting.

andrew-hirata-playeveryware commented on docs/steam/readme_steam.md:
```diff
@@ -0,0 +1,31 @@
+## External C# Wrappers for the Steam SDK
+
+The Steam SDK includes C++ libraries which needs C# wrappers to work in Unity.  
+There are several open source C# wrappers available.  
```
We should mention which ones we have samples for.

arthur740212 committed:
doc(steam) ; Fixes for PR

fixed : Too much info for adding identity provide, and add supported steamworks samples
